src/dashboard/dashboard.md

```diff
@@ -23,80 +24,93 @@
 
 More than just a repository for tokens, the TFChain Wallet is designed to offer a holistic financial experience. Users can securely store, send, and receive various digital assets while enjoying the benefits of deployable decentralized weblets built on the ThreeFold Grid. 
 
 Throughout this manual, we will explore the intricacies of the TFChain Wallet within the ThreeFold Dashboard, delving into its key features, security measures, and its role in facilitating decentralized financial activities.
 
-### Generate Account
+## Generate Account
 
 ![Connect](./img/dashboard_generate_account.png)
 ![Generate](./img/dashboard_T&C.png)
 
 Mnemonics serve as your private keys, acting as your representation on the ThreeFold Grid. You can input an existing mnemonic or click the **Generate Account** button to establish an account and generate a new mnemonic. In the latter case, you will be redirected to our **Terms and Conditions** page, where acceptance is required. Afterward, the mnemonic will be automatically created and filled into the form.
 
-### Connect Your Wallet
+## Connect Your Wallet
 
 ![Connect](./img/dashboard_connect.png)
 
 After entering your information, you can connect your wallet seamlessly.
 
-### Login
+## Login
 
 ![login](./img/dashboard_login.png)
 
 Once you've logged in with the password used while connecting your wallet, you can access all of the ThreeFold Dashboard sections.
 
-## Twin/Farm Management
+# Twin/Farm Management
 
-### Twin Management
+## Twin Management
 
 ![twin mgmt](./img/dashboard_portal_twin.png)
 
 The TF Twin management feature of the ThreeFold Dashboard enables users to create, manage, and monitor their individual digital entities known as **Twins**. A Twin can represent a virtual machine (VM) or a container running on the ThreeFold Grid. With the Twin management, users can easily deploy and scale their workloads, allocate resources, and configure networking and storage settings for their Twins.
 
-### Farms Management
+## Farms Management
 
 ![farm mgmt](./img/dashboard_portal_farms.png)
 
 The Farms management feature allows users to manage their own infrastructure, referred to as **Farms** on the ThreeFold Grid. Farms consist of servers and storage devices that contribute computing and storage capacity to the grid. Through the Farms management feature, users can add or remove, as well as monitor the health and performance of their farms, all the while efficiently using their resources.
+
+### Nodes Uptime
+
+Farmers can consult the Dashboard to see their 3Nodes' uptime. 
+
+To see the uptime of a given node, on the Dashboard go to the **Farms** section of the **Portal**, then under **Your Farm Nodes** click on the down arrow button situated on the left of the screen. On the right of the screen, you will see an icon with the uptime, as shown below:
+
+![ ](./img/dashboard_uptime.png)
+
+The uptime calculation is done as follows:
+
+> ((currentPeriod - downtime) / currentPeriod) * 100
```

Comment:
Can we replace the current period with the uptime (easier to read)?

Reply:
Good idea. Done in [last commit](https://github.com/threefoldtech/info_grid/pull/355/commits/f5e03e0be51249363b68d47b4ae506331d8e3bad)